Apollonius_graph_2/include/CGAL/Apollonius_graph_2/comparator_profiler.h

```diff
@@ -32,23 +33,22 @@ namespace ApolloniusGraph_2 {
 class comparator_profiler
 {
 public:
-  static bool count_cases;
-  static unsigned long case_1_counter;
-  static unsigned long case_2_counter;
-  static unsigned long case_3a_Jpos_counter;
-  static unsigned long case_3a_Jneg_counter;
-  static unsigned long case_3b_Jpos_counter;
-  static unsigned long case_3b_Jneg_counter;
-  static unsigned long case_4_counter;
-  static unsigned long case_5_counter;
-  static unsigned long case_degenerate_counter;
+  static CGAL::cpp11::atomic<bool> count_cases;
```

Comment:
The use of `CGAL::cpp11::atomic` must be protected by `#ifndef CGAL_NO_ATOMIC`: otherwise if the compiler does not use C++11, and Boost version is < 1.53, then it will not compile.